timeclock.py

```diff
@@ -158,11 +174,10 @@ def create_dev_users():
 
 
 @app.cli.command("create_health_screen_users")
-def create_dev_users():
+def create_health_screen_users():
     """Create health screen users for development."""
-
     faker = Faker()
-    for i in range(10):
+    for i in range(3):
```

Comment:
Creating 10 users feels like a lot. It's probably more representative to create a few users with carefully selected state (likes some timed in and some not, some with a history of check-ins already, etc) to make manual QA easier.